sdcm/utils/remote_logger.py

```diff
@@ -0,0 +1,135 @@
+        try:
+            self._log_retrieve(since)
+            self._retrieve(since)
+        except Exception as details:  # pylint: disable=broad-except
+            self._log.error('Error retrieving remote node DB service log: %s', details)
+
+    @raise_event_on_failure
+    def _journal_thread(self):
+        read_from_timestamp = None
+        while not self._termination_event.is_set():
+            self._wait_ssh_up(verbose=False)
+            self._retrieve_journal(since=read_from_timestamp)
+            read_from_timestamp = datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S")
+
+    def _wait_ssh_up(self, verbose=True, timeout=500):
+        text = None
+        if verbose:
+            text = '%s: Waiting for SSH to be up' % self
+        wait.wait_for(func=self._remoter.is_up, step=10, text=text, timeout=timeout, throw_exc=True)
+
+    def start(self):
+        self._thread.start()
+
+    def stop(self, timeout=None):
+        self._thread.terminate()
+        self._thread.join(timeout)
+        if self._thread.is_alive():
+            self._thread.kill()  # pylint: disable=no-member
+
+
+class SSHScyllaSystemdLogJournal(SSHLogJournalBase):
+    _logger_cmd = 'sudo journalctl -f --no-tail --no-pager --utc {since} ' \
+                  '-u scylla-ami-setup.service '\
+                  '-u scylla-image-setup.service '\
+                  '-u scylla-io-setup.service '\
+                  '-u scylla-server.service '\
+                  '-u scylla-jmx.service'
+
+
+class SSHScyllaFileLogJournal(SSHLogJournalBase):
+    _logger_cmd = 'sudo tail -f /var/log/syslog'
+    # _logger_cmd = 'sudo tail -f /var/log/syslog | grep scylla'
+
+
+class SSHCassandraFileLogJournal(SSHLogJournalBase):
+    _logger_cmd = 'sudo tail -f /var/log/cassandra/system.log'
+
+    def _retrieve(self, since):
+        wait.wait_for(self._file_exists, step=10, timeout=600, throw_exc=True,
+                      file_path='/var/log/cassandra/system.log')
+        super()._retrieve(since)
+
+
+class DockerLogJournal(LogJournalBase):
+    def __init__(self, node, target_log_file: str):
+        super().__init__(node, target_log_file)
+        self._logger_cmd = f'docker logs -f {node.name} >{target_log_file} 2>&1'
+        self._thread = None
+
+    def start(self):
+        self._thread = subprocess.Popen(self._logger_cmd, shell=True)
+
+    def stop(self, timeout=None):
+        if self._thread:
+            self._thread.kill()
+
+
+def get_system_logging_thread(logs_transport, node, target_log_file):
+    if logs_transport == 'docker':
+        return DockerLogJournal(node, target_log_file)
+    if logs_transport == 'ssh':
+        if node.init_system == 'systemd':
+            return SSHScyllaSystemdLogJournal(node, target_log_file)
+        elif node.file_exists('/usr/bin/scylla'):
+            return SSHScyllaFileLogJournal(node, target_log_file)
+        else:
+            return SSHCassandraFileLogJournal(node, target_log_file)
```

Comment:
I won't put it as fallback, not without the ability to test it

Reply:
selecting `SSHCassandraFileLogJournal` as a fallback seems weird, since we are not sure it's working, and no way of testing it.

I think we should remove those Cassandra parts out, since it's not working (not this specfic code, but the whole Cassandra support was used since I'm started working in scylla).